### PR TITLE
[aes] Fix lint errors related to unused clock and reset signals

### PR DESCRIPTION
**Pull request by vogelpi**
If fully combinational S-Box implementations are used, these signals are unused.

### CODE REVIEW

**Comment by vogelpi** [comment]
Hey @mwbranstad, could you please check if the errors indeed go away with this? I don't have access to the tool.

**Comment by mwbranstad** [comment]
@vogelpi as written, these waiver statements remove the lint errors from the lint checking step. It appears that you want to waive these, based on the implementation chosen. If that works for you, I am good with this change.

**Comment by vogelpi** [comment]
Cool, thanks for your feedback @mwbranstad ! Once this PR gets approved, I'll merge it.

**Comment by vogelpi** [comment]
Btw @mwbranstad , as you already looked at the PR, it would be nice if you could approve it. That means @msfschaffner or another reviewer with committer access doesn't need to check it again and can just approve it. Thanks.

**Review by mwbranstad — APPROVED** [review]
For implementation that does not use the clk_i or rst_ni inputs, the ascent lint error message can be safely waived.

**Review by msfschaffner — APPROVED** [review]
LGTM

**Pull request merged**
